cmd/virt-handler/virt-handler.go

```diff
@@ -306,10 +306,15 @@ func (app *virtHandlerApp) Run() {
 
 	stop := make(chan struct{})
 	defer close(stop)
+
+	virtHandlerNodeInformer := factory.KubeVirtSingleNode(app.HostOverride)
+	factory.Start(stop)
```

Comment:
I guess the `factory.Start()` call was moved here because the nodeLabellerController is started a few lines down?
Can we, instead, move the `nodeLabellerController.Run()` down, where the vmController is activated? This will allow us to keep the `factory.Start()` call in a separate phase, after all the controllers and their required informers have been created and configured.

Reply:
Changed.